src/main/java/com/cinema/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.cinema.dao.impl;
+
+import com.cinema.dao.MovieSessionDao;
+import com.cinema.exception.DataProcessingException;
+import com.cinema.lib.Dao;
+import com.cinema.model.MovieSession;
+import com.cinema.util.HibernateUtil;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class MovieSessionDaoImpl implements MovieSessionDao {
+    @Override
+    public List<MovieSession> findAvailableSessions(Long movieId, LocalDate date) {
+        LocalDateTime fromDateTime = LocalDateTime.of(date, LocalTime.of(0,0));
+        LocalDateTime toDateTime = LocalDateTime.of(date, LocalTime.of(23,59, 59));
```

Comment:
let's use LocalDate methods instead of hardcoding this

Reply:
done